docs/RELEASE.md

```diff
@@ -43,70 +83,47 @@ $ python setup.py sdist bdist_wheel
 [`MANIFEST.in`](../MANIFEST.in) determines which non-Python files are included.
 Make sure to check that all necessary ones are listed before beginning the release process.
 
-## uploading to PyPI
+## uploading the release candidate to PyPI
 
 You'll need `twine` installed for this step:
 ```bash
 $ pip install twine
 ```
 
-To make sure that everything is working properly, first upload to `test.pypi.org`,
-entering your credentials when prompted:
+Upload the release candidate with:
 ```bash
-$ python -m twine upload --repository-url=https://test.pypi.org/legacy/ dist/*
+$ python -m twine upload dist/*
```

Comment:
The dist directory might stick around, so it's better to explicitly point to the right file here.

Reply:
would that be something like
```bash
$ python -m twine upload dist/napari-0.2.5.tar.gz
```
Can I just ignore `napari-0.2.5-py3-none-any.whl` or do I have to point at that too? (these are the two files in the directory)

Reply:
Possibly it's good to upload the wheel also, since it is universal (any). I think it makes install a bit faster for users.

Having said this, it occurs to me that there should maybe be something there about the Python version...? Safest would be to upload only the `.tar.gz` until we can figure that out...

Reply:
ok, i will update to `$ python -m twine upload dist/napari-X.Y.Z.tar.gz` and `$ python -m twine upload dist/napari-X.Y.Zrc1.tar.gz` in the appropriate places, we can always test with our next release candidate and see!!

Reply:
fixed